plenum/simulation/hyp_instance_change.py

```diff
@@ -0,0 +1,61 @@
+from hypothesis import strategies as st, reproduce_failure
+from hypothesis import given, settings
+
+from plenum.simulation.pool_model import RestartEvent, OutageEvent, CorruptEvent, PoolModel
+from plenum.simulation.sim_event_stream import ListEventStream, sim_events, ErrorEvent
+from plenum.simulation.sim_model import ModelWithExternalEvents, process_model
+from plenum.simulation.timer_model import TimerModel
+
+settings.register_profile("big", buffer_size=128 * 1024, max_examples=1000)
+settings.load_profile("big")
+
```

Comment:
Agreed, although if at some point in future we get approval to add Hypothesis as test dependency then this can be reduced to just two separate modules:
- simulation helper classes
- tests for simulation helper classes

And real simulation tests can be put next to other tests related to tested part of production code